content/hi/docs/setup/production-environment/_index.md

```diff
@@ -0,0 +1,135 @@
+- *उपयोगकर्ता प्रमाणपत्र और रोल बाइंडिंग (RBAC) बनाएं*: यदि आप आरबीएसी प्राधिकरण का उपयोग कर रहे हैं, तो उपयोगकर्ता एक सर्टिफिकेट साइनिंग रिक्वेस्ट (सीएसआर) बना सकते हैं जिस पर क्लस्टर सीए द्वारा हस्ताक्षर किए जा सकते हैं। फिर आप प्रत्येक उपयोगकर्ता के लिए रोल्स और क्लस्टररोल्स को बाध्य कर सकते हैं।
+विवरण के लिए [प्रमाणपत्र पर हस्ताक्षर करने के अनुरोध](/docs/reference/access-authn-authz/certificate-signing-requests/) देखें।
+- *ऐसी नीतियां बनाएं जो विशेषताओं को जोड़ती हैं (ABAC)*: यदि आप एबीएसी प्राधिकरण का उपयोग कर रहे हैं, तो आप चुनिंदा उपयोगकर्ताओं या समूहों को विशेष संसाधनों (जैसे पॉड), नेमस्पेस, या एपीग्रुप तक पहुंचने के लिए अधिकृत करने के लिए नीतियां बनाने के लिए विशेषताओं के संयोजन असाइन कर सकते हैं। अधिक जानकारी के लिए देखें [उदाहरण](/docs/reference/access-authn-authz/abac/#examples)।
+- *प्रवेश नियंत्रकों पर विचार करें*: एपीआई सर्वर के माध्यम से आने वाले अनुरोधों के लिए प्राधिकरण के अतिरिक्त रूपों में शामिल हैं
+[वेबहुक टोकन प्रमाणीकरण](/docs/reference/access-authn-authz/authentication/#webhook-token-authentication)। वेबहुक और अन्य विशेष प्राधिकरण प्रकारों को एपीआई सर्वर में [प्रवेश नियंत्रक](/docs/reference/access-authn-authz/admission-controllers/) जोड़कर सक्षम करने की आवश्यकता है।
+
+## कार्यभार संसाधनों पर सीमा निर्धारित करें
+
+उत्पादन कार्यभार की मांग कुबेरनेट्स नियंत्रण विमान के अंदर और बाहर दोनों जगह दबाव पैदा कर सकती है। अपने क्लस्टर के वर्कलोड की जरूरतों के लिए सेट अप करते समय इन मदों पर विचार करें:
+
+- *नेमस्पेस्स की सीमा निर्धारित करें*: मेमोरी और सीपीयू जैसी चीजों पर प्रति नेमस्पेस कोटा सेट करें। विवरण के लिए [मेमोरी, सीपीयू और एपीआई संसाधन प्रबंधित करें](/docs/tasks/administer-cluster/manage-resources/) देखें।
+आप इनहेरिट करने की सीमा के लिए [हिरार्चीकैल नेमस्पेस्स](/blog/2020/08/14/introducing-hierarchical-namespaces/) भी सेट कर सकते हैं।
+- *DNS मांग के लिए तैयार करें*: यदि आप वर्कलोड के बड़े पैमाने पर बढ़ने की उम्मीद करते हैं, तो आपकी DNS सेवा भी बड़े पैमाने पर तैयार होनी चाहिए।
+[क्लस्टर में DNS सेवा को ऑटोस्केल करना](/docs/tasks/administer-cluster/dns-horizontal-autoscaling/) देखें।
+- *अतिरिक्त सेवा खाते बनाएं*: उपयोगकर्ता खाते यह निर्धारित करते हैं कि उपयोगकर्ता क्लस्टर पर क्या कर सकते हैं, जबकि एक सेवा खाता किसी विशेष नेमस्पेस्स के भीतर पॉड एक्सेस को परिभाषित करता है। डिफ़ॉल्ट रूप से, एक पॉड अपने नेमस्पेस्स से डिफ़ॉल्ट सेवा खाता लेता है।
+नया सेवा खाता बनाने के बारे में जानकारी के लिए [सेवा खातों का प्रबंधन](/docs/reference/access-authn-authz/service-accounts-admin/) देखें।
+उदाहरण के लिए, आप शायद यह करना चाहें:
+  - ऐसे रहस्य जोड़ें जिनका उपयोग पॉड किसी विशेष कंटेनर रजिस्ट्री से इमेज खींचने के लिए कर सकता है। उदाहरण के लिए [पॉड्स के लिए सेवा खाते कॉन्फ़िगर करें](/docs/tasks/configure-pod-container/configure-service-account/) देखें।
+  - किसी सेवा खाते में RBAC अनुमतियाँ असाइन करें। विवरण के लिए [सेवा खाता अनुमतियां](/docs/reference/access-authn-authz/rbac/#service-account-permissions) देखें।
+
+## {{% heading "आगे क्या है" %}}
```

Comment:
```suggestion
## {{% heading "whatsnext" %}}
```

Reply:
See https://github.com/kubernetes/website/pull/29518#discussion_r785278840

Reply:
@keshavcodex If you can squash the commits after committing the above suggestions, please do